network/ServerNetworking.h

```diff
@@ -227,7 +227,7 @@ class PlayerConnection :
     const std::string& ClientVersionString() const;
 
     /** Returns true iff the client's specified version string matches the
-      * version string of this server process. */
+      * version string of this server process and the client could accept binary serialization. */
```

Comment:
Likewise, since the added option is purely server-side, the change does not seem to have to do with whether the client could accept binary serialization -- it looks to me like the initial messages establishing the connection and sharing the client version number are done via xml serialization, and I don't see that there has been any check yet done at this point to confirm that the client really could exchange compatible binary messages.  Or am I missing/misreading something?

Reply:
I've added some description because this function is used only to enable binary serialization when sending turn update and game start messages.

Reply:
I am not understanding how that really relates to my concern -- it seems to me like the code comment change here indicates that the function is checking whether "the client could accept binary serialization", but it does not look to me like the code actually checks that in any way.

Reply:
Tried to rephrase it.